src/UI/Factory.php

```diff
@@ -22,10 +22,14 @@
 	 *     They indicate the functionality or nature of a text-element or context:
 	 *     Icons will mainly be used in front of object-titles, e.g. in the
 	 *     header, the tree and in repository listing.
+	 *     Icons can be disabled. Disabled Icons visually communicate that they
+	 *     can not be worked with by users, i.e. a precondition is not met yet or
```

Comment:
I do not think, that the "icons" are the things that the user can work with. It rather is the indicated funtionality that the user cannot use.

Reply:
Maybe rephrase to: "... communicate that the depicted functionality is not available to the user."